### DISCUSSION TITLE
Specifying external libs

### DISCUSSION THREAD

**Discussion by mrgoldair** [zed-industries/zed#6642] [Feature Requests]
I have SDL2 installed via homebrew but I'm still getting red squiggles everywhere. I've looked at all the config options but I can't find anything like `includePath` from VSCode where I can specify the SDL2 libs.

<img width="1624" alt="image" src="https://github.com/zed-industries/community/assets/61611125/350bcda8-7cac-428e-845f-e955184d38a1">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
